src/test/java/org/apache/commons/lang3/builder/ToStringBuilderTest.java

```diff
@@ -316,7 +316,7 @@ public void testReflectionHierarchyArrayList() {
         // representation different for IBM JDK 1.6.0, LANG-727
         assumeFalse("IBM Corporation".equals(SystemUtils.JAVA_VENDOR) && "1.6".equals(SystemUtils.JAVA_SPECIFICATION_VERSION));
         assumeFalse("Oracle Corporation".equals(SystemUtils.JAVA_VENDOR) && "1.6".compareTo(SystemUtils.JAVA_SPECIFICATION_VERSION) < 0);
-        final List<Object> list = new ArrayList<>();
+        final List<Object> list = new ArrayList<>(10);
```

Comment:
I think that needs a comment.
Is the magic number 10 significant?
If so, what determines the value?
Could it ever change?